src/Initializer/Template/SymfonyFlexTemplate.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/* (c) github.com/zorn-v
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Deployer\Initializer\Template;
+
+/**
+ * Generate a Symfony flex deployer configuration.
+ *
+ * @author zorn-v
+ * @codeCoverageIgnore
+ */
+class SymfonyFlexTemplate extends FrameworkTemplate
+{
+    protected function getRecipe()
+    {
+        return 'symfony-flex';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExtraContent()
+    {
+        return <<<PHP
+
+// Migrate database before symlink new release.
+
+before('deploy:symlink', 'database:migrate');
+
+//If you don't use DotEnv in prod just wipe it out
+task('clear:env', function () {
```

Comment:
What purpose of this task?

Reply:
DotEnv does not run at all if APP_ENV is set https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/public/index.php#L11

Reply:
I think it's better not to hack APP_ENV but unstead write comment what user need to move dotenv to require section.

Reply:
But this task needed exactly in order if DotEnv in require ;)
One more time - DotEnv does not run if APP_ENV is set.
If you don't set it in env, `composer install` will fail.

Yes I know that is hack. But I think that I can not convince community to change `!isset($_SERVER['APP_ENV'])` to `!file_exists(__DIR__.'/../.env')`